src/app/highlight-cards/HighlightCard.spec.ts

```diff
@@ -112,23 +112,6 @@ describe('highlightCard', () => {
     expect(highlightCardForHomepage.button.href.href).toBe('https://example-blog.com/some-path');
   });
 
-  it('should replace CC24 link with metacampaign page', () => {
```

Comment:
This will be set correctly at source after https://github.com/thebiggive/salesforce/pull/2175 is out, no more rewrites.